doozer/doozerlib/cli/scan_sources.py

```diff
@@ -54,6 +68,143 @@ def run(self):
         # We have our information. Now build and print the output report
         self.generate_report()
 
+    def _try_reconciliation(self, metadata: Metadata, repo_name: str, branch_name: str):
+        reconciled = False
+
+        # Attempt a fast-forward merge
+        rc, _, _ = exectools.cmd_gather(cmd=['git', 'pull', '--ff-only', 'public_upstream', branch_name])
+        if not rc:
```

Comment:
`rc` is a number. let's not use it like a boolean.

Reply:
In Python it's broadly accepted (and even suggested) to run such checks against non-boolean types such as strings, lists, and objects in general. I'd like to leave this as-is. Besides, we use the same pattern everywhere across our code.